assignments/resume-bootstrap/resume.html

```diff
@@ -0,0 +1,229 @@
+                        <ul id="ulE">
+                            <li>Majored in Information Technology</li>
+                            <li>Minored in Decision Support</li>
+                        </ul>
+
+                        <h5>Other courses</h5>
+                        <ul class="ulE2">
+                            <li>CCNA</li>
+                            <li>Android</li>
+                            <li>Arduino</li>
+                            <li>Windows Azure</li>
+                            <li>Cloud Computing </li>
+                        </ul>
+                    </article>
+
+                    <article>
+                        <h2>Projects</h2>
+                        <div class="maindot"></div>
+                        <h3>Digital Watermarking</h3>
+                        <ul class="ulp">
+                            <li>Java application that can hide Text inside a RGB picture using Least Significant Bit (LSB) Algorithm
+                                and encrypt it with password.
+                            </li>
+                        </ul>
+                        <h3>Reverse Game</h3>
+                        <ul class="ulp">
+                            <li>Is a board game, I created it with Java and store Score on Java Server (Server Socket) with JDBC
+                                & MYSQL.</li>
+                        </ul>
+                        <h3>MagPro (Android Application)</h3>
+                        <ul class="ulp">
+                            <li>Android App to manage Project and find workers with specific needs.</li>
+                        </ul>
+                        <h3>Contact manager</h3>
+                        <ul class="ulp">
+                            <li>Algorithm to store and retrieve names based on tree algorithm.
+                            </li>
+                        </ul>
+                        <h3>Make website using python and django:</h3>
+                        <ul class="ulp">
+                            <li>The user can enter the temperature and humidity for today and can see the history for all previous
+                                days and can see the summary for every 10 days. </li>
+                        </ul>
+                        <h3>Two Websites</h3>
+                        <ul class="ulE2">
+                            <li> using (“HTML , CSS , Java Script , PHP,MYSQL” , “ASP.NET” ) .</li>
+                        </ul>
+                    </article>
+                </div>
+                <div class="col-xs-12 col-md-6 col-sm-6" id="rightcontent">
+                    <article>
+                        <h2>Skills & Competences</h2>
+                        <div class="maindot"></div>
+                        <h3>Computer Skills</h3>
+                        <ul class="uls">
+                            <li>
+                                <p>java</p>
+                                <progress value="90" max="100"></progress>
```

Comment:
Consider using `<meter>` instead of `<progress>` because it is semantically more accurate.